iscript/src/iscript/mac.py

```diff
@@ -279,11 +291,13 @@ async def sign_app(sign_config, app_path, entitlements_path, provisioning_profil
 
         for file_ in files:
             abs_file = os.path.join(top_dir, file_)
+            sign_command = _get_sign_command(identity, keychain, sign_config, file_=file_, entitlements_path=entitlements_path)
             await _do_sign_file(top_dir, abs_file, file_, sign_command, app_path_len, app_executable)
 
-    await sign_libclearkey(contents_dir, sign_command, app_path)
+    await sign_libclearkey(contents_dir, _get_sign_command(identity, keychain, sign_config), app_path)
```

Comment:
Does this mean we no longer sign libclearkey with entitlements?

Reply:
(Did we check to make sure Firefox still starts up with this patch applied?)

Reply:
I did! But I'm going to do some log comparison to make sure signing command invocations haven't changed vs. current deployed code.

Reply:
Indeed! Thanks for catching this. Firefox still runs with it, but I imagine this would mean Widevine would break.